public/sass/custom/kth-klaro.scss

```diff
@@ -67,58 +50,42 @@
       #id-cookie-notice {
         color: $black;
       }
-      .cn-ok {
-        flex-flow: column !important;
-      }
-      .cm-link {
-        &.cn-learn-more {
-          color: #036eb8;
-          margin-bottom: 0.625rem;
-          text-decoration: underline;
-          text-underline-offset: 0.25rem;
-        }
-      }
+
       .cn-buttons {
+        margin-top: 1rem;
+
         @media (max-width: 575px) {
           margin-top: 0.5rem;
         }
-        margin-top: 1rem;
 
         button {
           &.cm-btn {
             @media (max-width: 384px) {
               width: auto;
             }
 
-            &.cm-btn-success {
-              color: $white;
-              background-color: #004791;
-              border: 1px solid transparent;
+            &.cm-btn-success,
+            &.cm-btn-danger.cn-decline,
+            &.cm-btn-learn-more.cm-btn-info {
               padding: 0.5rem 1.5rem;
-              border-radius: 0.25rem;
               min-height: 2.75rem;
               font-weight: bold;
               font-size: 1rem;
+              border-radius: 0.25rem;
             }
 
-            &.cm-btn-danger {
-              &.cn-decline {
-                min-height: 2.75rem;
-                color: $black;
-                background-color: $white;
-                border: 1px solid $black;
-                font-weight: bold;
-                padding: 0.5rem 1.5rem;
-
-                @media (max-width: 575px) {
-                  margin-bottom: 0.5rem;
-                }
-              }
+            &.cm-btn-success {
+              color: $white;
+              background-color: #004791;
+              border: 1px solid transparent;
             }
-            @media (max-width: 575px) {
-              &.cm-btn-success {
-                margin-top: 0.5em;
-              }
+
+            &.cm-btn-danger.cn-decline,
+            &.cm-btn-learn-more.cm-btn-info {
+              color: $black;
+              background-color: $white;
+              border: 1px solid $black;
+              margin-bottom: 1rem;
```

Comment:
Why do these have bottom margin, but not success?

Reply:
I thought that adding a margin-bottom might be necessary when the buttons shift position on smaller screens. I was considering using a media query instead, but I'm not sure when the buttons move to a block layout.

Reply:
Typically we don't want to add that type of logic to the child elements, but rather to the parent since this will break if we would rearrange the order of the buttons, add/remove a button, etc. I would recommend adding `display: flex; flex-wrap: wrap; gap: 0.5rem;` to `cn-buttons` and then remove all type of margins for all buttons.